core-bundle/contao/templates/backend/be_tinyMCE.html5

```diff
@@ -16,26 +16,29 @@ window.tinymce && tinymce.init({
   document_base_url: '<?= Environment::get('base') ?>',
   entities: '160,nbsp,60,lt,62,gt,173,shy',
   branding: false,
+  promotion: false,
   skin: 'tinymce-5',
-  setup: function(editor) {
+  setup: editor => {
     editor.getElement().removeAttribute('required');
+    document.querySelectorAll('[accesskey]').forEach(el => {
+      editor.addShortcut('access+' + el.accessKey, el.id, () => el.click());
+    });
   },
-  init_instance_callback: function(editor) {
+  init_instance_callback: editor => {
+    console.log(document.activeElement.id);
```

Comment:
that's probably a leftover?

Reply:
Yes. Removed in 9819d6cfbba5771b096a646f6199792d92a7d0a8.